packages/cli/src/Generate.ts

```diff
@@ -234,8 +234,21 @@ ${chalk.dim('```')}
 ${highlightTS(`\
 import { PrismaClient } from '${importPath}'
 const prisma = new PrismaClient()`)}
-${chalk.dim('```')}${breakingChangesStr}${versionsWarning}`
+${chalk.dim('```')}${
+          prismaClientJSGenerator.options?.dataProxy
+            ? `
+
+To use Prisma Client in edge runtimes like Cloudflare Workers or Vercel Edge Functions, import it like this:
+${chalk.dim('```')}
+${highlightTS(`\
+import { PrismaClient } from '${importPath}/edge'`)}
+${chalk.dim('```')}
+You will need a Prisma Data Proxy connection string. See documentation: ${link('https://pris.ly/d/data-proxy')}
```

Comment:
This applies to the standard import as well when generated via `--data-proxy`, right?

Reply:
I believe so, yes. Should we add an empty line before the sentence to separate it from the edge import and make it a paragraph on its own?

Reply:
@aqrln This  `prismaClientJSGenerator.options?.dataProxy ? 'your added' text` makes it look like it will only be added if `dataProxy` is truthy. So only adding the new text if true?

Reply:
@Jolg42 yes, correct. We only print this text when generating the pair of Data Proxy clients with `prisma generate --data-proxy`, the message is unchanged for the regular client.

Reply:
```suggestion

You will need a Prisma Data Proxy connection string. See documentation: ${link('https://pris.ly/d/data-proxy')}
```